src/main/java/com/appium/manager/TestLogger.java

```diff
@@ -232,14 +232,14 @@ private void handleTestFailure(ITestResult result, String className,
                 System.out.println("************************" + framedImageIOS.exists()
                         + "***********************");
                 if (framedImageIOS.exists()) {
-                    log.addScreenCaptureFromPath("screenshot/iOS/"
+                    log.addScreenCaptureFromPath(System.getProperty("user.dir") + "/target/screenshot/iOS/"
```

Comment:
Just tried now. Using relative path, extentx is not working. The extentx is only work when we use the statement - ExtentTestManager.getTest().addScreenCaptureFromPath(System.getProperty("user.dir") + "/target/screenshot/iOS/"...... Can I update this pull request for the same.

Also note that ExtentCucumberFormatter. java contains the same statement.